unime/src/routes/(app)/scan/+page@.svelte

```diff
@@ -141,7 +145,14 @@
           </div>
         {/if}
 
-        {#if $state?.dev_mode !== 'Off'}
+        {#if loading}
```

Comment:
It was a little unexpected that the "Scan a QR Code" heading disappears and then with a delay the spinner on an almost empty screen. I would swap out the camera portion only and overlay the spinner there.